frontend/src/components/Volunteer/volunteer.jsx

```diff
@@ -8,195 +8,68 @@ class Volunteer extends Component {
     super(props);
 
     this.state = {
-      role: "",
-      name: "",
-      email: "",
-      phone: "",
-      website: "",
-      message: ""
+      message: "",
```

Comment:
message can be removed as well

Reply:
Good catch. I made the change and pushed the commit.